src/main/java/uk/gov/companieshouse/confirmationstatementapi/contoller/StatementOfCapitalController.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.confirmationstatementapi.contoller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ServiceException;
+import uk.gov.companieshouse.confirmationstatementapi.model.StatementOfCapital;
+import uk.gov.companieshouse.confirmationstatementapi.service.StatementOfCapitalService;
+
+@RestController
+@RequestMapping("/transactions/{transaction_id}/confirmation-statement")
+public class StatementOfCapitalController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfirmationStatementController.class);
+
+    private final StatementOfCapitalService statementOfCapitalService;
+
+    @Autowired
+    public StatementOfCapitalController(StatementOfCapitalService statementOfCapitalService) {
+        this.statementOfCapitalService = statementOfCapitalService;
+    }
+
+    @GetMapping("/{companyNumber}/statement-of-capital")
```

Comment:
Why do we need this endpoint?

Reply:
To get the statement of capital data to the web for display on the page. It is the api that call the oracle query api so it has to go through this endpoint.

Reply:
Why can't the web call the oracle api directly?
This just seems to be adding a middleman api request but i'm not sure that it's adding any value

Reply:
I was hoping to avoid it but had a discussion with Steve and we need the api to be able access this data.

Reply:
We need a discussion by the sounds of it.

Reply:
spoke with Steve happy for it to go in, but could you add this endpoint to the swagger spec first